tasks/release.js

```diff
@@ -15,6 +15,7 @@ const dirsToCopy = [
 	'dist',
 	'i18n',
 	'images',
+	'vendor',
```

Comment:
Used to release.

Reply:
Absolutely. The `dev` packages add **a lot** of bloat. [Here's how we do it](https://github.com/Automattic/woocommerce-payments/blob/develop/package.json#L22) in WCPay, we make sure that the task that's run when we're preparing the release executes `composer install --no-dev`.

Reply:
Done in 573a36379a0555908681a70320d946373835b59d